iwyu_include_picker.cc

```diff
@@ -1301,12 +1307,9 @@ vector<IncludePicker::MappedInclude> IncludePicker::GetPublicValues(
   if (!values || values->empty())
     return retval;
 
-  if (GetOrDefault(filepath_visibility_map_, key, kPublic) == kPublic)
```

Comment:
This was subsumed by the new code in `IncludePicker::GetCandidateHeadersForFilepath` under the "We also need to consider the header itself" comment.  It makes more sense to put it there, because here the key might be a symbol, a full path, or a quoted include.  Only one of those makes sense to add to `retval`.